src/frontend/src/components/tables/company/AddressTable.tsx

```diff
@@ -108,6 +108,52 @@ export function AddressTable({
     ];
   }, []);
 
+  const addressFields: ApiFormFieldSet = {
```

Comment:
Not sure about the performance, have you tested if there is a difference if you put this in a usememo? I think then the entire form doesn't get rerendered when any other state eg for the table on this component changes.

Reply:
If there were anything in `addressFields` which was dynamic, I would put it in a `useMemo()` - but I would assume that as this is a static variable, tagging it as `const` is sufficient? Happy to be told otherwise :)

Reply:
That's only partially correct. If we pass the fields inside the form component anywhere into a dependency array, this will constantly rerun, as fields will be different every component render. However if we put it into a useMemo with an empty dependency array, it always returns the same object, so the fields is not responsible for a rerun of the hook. That's also the reason to why we sometimes need a useCallback.

Reply:
Ah, ok.. is it worth the extra boilerplate to wrap this in a `useMemo` for the performance boost, in your opinion?

Reply:
In my opinion yes, but it's not that much to add, just two lines.